Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -380,77 +395,92 @@ public void JumpCheck()
 				jumpColCheckTimer = 0.0f;
 				//actor.GetAnimator().SetBool("isSliding", false);
 			}
-			else if ( jumpColCheckTimer > jumpColCheckTime && isOnGround && !IsInState( ActorStates.Rolling ) )
-			{
-				if ( !IsInState( ActorStates.Grounded ) )
-				{
-					// if !launching
-					ChangeState( ActorStates.Grounded );
-					stopMoveTimer = 0f;
-					//GainControl();
-				}
-
-				actor.animator.SetBool( "isJumping", false );
```

Comment:
There's a big logic chunk that got cut off. We basically need to put logic back in to control animation states somewhere since it's taken out here.